.github/workflows/stale.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Stale Issue and PR Cleanup'
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 4 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4.1.0
+        id: stale
+        with:
+          days-before-stale: 60
+          days-before-close: 14
+          days-before-pr-close: 30
+          exempt-all-issue-allignees:
+          exempt-draft-pr:
+          stale-issue-label: Pending closure
+          only-pr-labels: '0 - Waiting on User'
+          close-issue-label: Unresolved
+          close-pr-label: 'Unresolved'
+          exempt-issue-labels: 'Security / CVE,0 - Backlog,1 - Ready for work,2 - Working, 3 - Review, 5 - Push required'
+          exempt-pr-labels: 'Security / CVE'
+          labels-to-remove-when-unstale: '0 - Wating on User,Pending closure'
+          stale-issue-message: |
+            Is this still relevant? If so, what is blocking it? Is there anything you can do to help move it forward?
+            This issue will be closed in 14 days if it continues to be inactive.
+          close-issue-message: |
+            Dear contributor,
+
+            because this issue seems to have been inactive for quite some time now, I've automatically closed it.
```

Comment:
```suggestion
            Due to the fact that this issue seems to have been inactive for quite some time now, I've automatically closed it.
```

Reply:
Can I suggest 'As' instead of 'Due to the fact'. Sounds a bit 'softer'.

Reply:
Not sure I follow. Mean something like `As this issue seems...`?

Reply:
Yes, I believe that is what @pauby is suggesting. I have no objections to this.